src/components/flat-table/flat-table.mdx

```diff
@@ -287,3 +296,19 @@ The `FlatTableRow` relies on specific child composition to identify sticky colum
 ### FlatTableCheckbox
 
 <ArgTypes of={FlatTableCheckboxStories} />
+
+## Translation keys
+
+The following keys are available to override the translations for this component by passing in a custom locale object to the
+[i18nProvider](https://carbon.sage.com/?path=/story/documentation-i18n--page).
```

Comment:
**issue**: this URL won't work unfortunately due to a change in Storybook v7 with link generation. Could we change this to the following please?: 

```suggestion
[i18nProvider](../?path=/docs/documentation-i18n--docs).
```